src/rofi_rbw/selector.py

```diff
@@ -117,5 +140,8 @@ def show_selection(
             capture_output=True,
             encoding='utf-8'
         )
-        return wofi.returncode, wofi.stdout
+        returnaction = None
```

Comment:
I'm not a fan of this pattern where one sets a variable, conditionally changes it, and then return it. Instead, I'd suggest a simpler
```
if wofi.returncode == 0:
  return default_action, wofi.stdout
else
  return None, wofi.stdout
```

Reply:
Alright, i'll change it!